autoware-nightly.repos

```diff
@@ -15,6 +15,10 @@ repositories:
     type: git
     url: https://github.com/autowarefoundation/autoware_utils.git
     version: main
+  core/autoware_rviz_plugins:
```

Comment:
I'm not sure if we should put this under `core` or we should create a new directory like `visualization`.
Do you have any ideas @xmfcx?